app/Db/pu610k.db

```diff
@@ -48,6 +48,28 @@ record(bo, "$(BASE):$(CHAN):ENABLE_RBV") {
     field(ONAM, "Enabled")
 }
 
+record(calc, "$(BASE):CHILLER_OK") {
+    field(DESC, "LPL Glass Rod chillers on")
+    field(INPA, "$(CHILL):1")
+    field(INPB, "$(CHILL):2")
+    field(INPC, "$(CHILL):3")
+    field(PINI, "YES")
```

Comment:
Picking nits: If you're scanning the record, then `PINI` is not needed.

Reply:
add CPP to these.